Bundle/CoreBundle/Resources/public/js/edit/widget.js

```diff
@@ -3,42 +3,71 @@
 
 
 // Open select when click on dropdown link
-$vic(document).on('focus', '.vic-new-widget select', function(event) {
-    $vic(this).addClass('vic-open');
+$vic(document).on('focus', '.v-slot select', function(event) {
+    $vic(this).parent('.v-slot').addClass('v-slot--open');
 });
-$vic(document).on('blur', '.vic-new-widget select', function(event) {
+$vic(document).on('blur', '.v-slot select', function(event) {
+    $vic(event.target).parent('.v-slot').removeClass('v-slot--open');
     $vic(event.target).prop('selectedIndex', 0);
-    $vic(event.target).removeClass('vic-open');
 });
 
+// Eval width of the slots to add or not a class
```

Comment:
because we want to set the 'small slot mode' depending of the size of the slot container and not on the media-query. If the viewport is wide, but the slot is inside a small column, we want it to be in 'small mode'